src/components/wallet/WalletManage.tsx

```diff
@@ -43,6 +44,9 @@ const WalletManage: React.FC = (): JSX.Element => {
   const [network, setNetwork] = useState<string | null>()
   const [chainId, setChainId] = useState<string | null>()
   const [address, setAddress] = useState<string | null>('')
+
+  const { lock, phrase } = useWalletContext()
+
   async function setData() {
```

Comment:
How about using arrow function?
```const setData = async () => {```

Reply:
Good point! Since that's legacy code (you can't know it) and `WalletManage` has to be refactored in either way, we will do it then.